File: HLCObjKotlinAct4/src/Java/Act13/Anime.java
package Java.Act13;

public class Anime {
    String nombre;
    int episodios;
    String genero;

    public Anime(String nombre, int episodios, String genero) {
        this.nombre = nombre;
        this.episodios = episodios;
        this.genero = genero;
    }

    @Override
    public String toString() {
        return nombre + "\n" +
                "Numero de episodios: " + episodios + "\n"
                + "Genero: " + genero;
    }

}

File: HLCObjKotlinAct4/src/Kotlin/Act13/Anime.kt
package Kotlin.Act13

class Anime(val nombre: String?,
            val episodios: Int,
            val genero: String?) {

    override fun toString(): String {
        return (nombre + "\n" +
                "Numero de episodios: " + episodios + "\n"
                + "Genero: " + genero)
    }
}
